zebra-rpc/src/methods/types/get_raw_mempool.rs

```diff
@@ -0,0 +1,116 @@
+use std::collections::HashMap;
```

Comment:
```suggestion
//! Types used in `getrawmempool` RPC method.

use std::collections::HashMap;
```

https://github.com/ZcashFoundation/zebra/blob/2cf84e3add97594114486b88bba3396ee5952bdb/zebra-rpc/src/methods/types/get_blockchain_info.rs#L1